src/main/java/gate/creole/annic/lucene/LuceneSearchThread.java

```diff
@@ -47,6 +48,13 @@
    */
   private static boolean DEBUG = false;
 
+  private static final XStream xstream;
+  
+  static {
+	  xstream = new XStream(new StaxDriver());
+	  Gate.configureXStreamSecurity(xstream);
```

Comment:
XStream security.. think the same thing is done in seal commons.

Reply:
Ok